content/v2.0/influxdb-templates/create.md

```diff
@@ -79,6 +79,21 @@ influx pkg export all \
   -t $INFLUX_TOKEN
 ```
 
+#### Export resources filtered by labelName or resourceKind
+The `influx pkg export all` command has an optional `--filter` flag that filters
+only exports resources that match a specified `labelName` or `resourceKind`.
```

Comment:
I think it'd be worth mentioning that all like filter types are `OR`ed together and `AND`ed across the the grouping of both.

An exampe to illustrate:

```sh
influx pkg export all \
  -o my-org \
  -f ~/templates/awesome-template.yml \
  -t $INFLUX_TOKEN \
  --filter=resourceKind=Bucket \
  --filter=resourceKind=Dashboard \
  --filter=labelName=Example1 \
  --filter=labelName=Example2
```

the above would return resources that match the following criteria:

```
(labelName == "Example1" || labelName == "Example2") && (resourceType == Bucket || resourceType == Dashboard)
```